backend/src/users/controllers.ts

```diff
@@ -126,7 +126,7 @@ const getUsers = async (
   // Handles GET /users?limit=20&after=asdf
   let cursor = undefined;
   let skip = undefined;
-  if (pagination.after) {
+  if (pagination.after != "") {
     cursor = {
       id: pagination.after as string,
     };
```

Comment:
this is to address the case of passing an empty cursor, if an empty cursor is passed, the backend should be able to ignore it.